src/popover/container.tsx

```diff
@@ -91,24 +94,22 @@ export default function PopoverContainer({
       width: parseFloat(getComputedStyle(arrow).width),
       height: parseFloat(getComputedStyle(arrow).height),
     };
-    const containingBlock = getContainingBlock(body);
+    const containingBlock = getContainingBlock(popover);
     const containingBlockRect = containingBlock ? containingBlock.getBoundingClientRect() : viewportRect;
 
-    // Round up dimensions (IE11 doesn't handle subpixels too accurately)
-    const bodyRect = body.getBoundingClientRect();
-    const bodyRectCeil = {
-      top: bodyRect.top,
-      left: bodyRect.left,
-      width: Math.ceil(bodyRect.width),
-      height: Math.ceil(bodyRect.height),
-    };
+    const contentRect = contentRef.current.getBoundingClientRect();
+    const contentBoundingBox = { width: contentRect.width, height: contentRect.height };
```

Comment:
The measurements have to be performed on the actual content which dimensions stay unchanged. If measure the popover dimensions instead - those will change once scrolling is allowed which can introduce an infinite update loop:

1. The popover can't fit on the screen;
2. The position is calculated and the popover body made scrollable;
3. The change to the popover body height triggers resize observer and new position calculation;
4. The new popover size can now fit the screen so that scrollable flag is removed;
5. Back to 1.